_ml-commons-plugin/api/model-apis/batch-predict.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: default
+title:  Batch inference
+parent: Model APIs
+grand_parent: ML Commons APIs
+nav_order: 20
+---
+
+# Batch inference
+
+ML Commons can predict large datasets in an offline asynchronous mode with your remote model deployed in external model servers. To use the Batch_Predict API, the `model_id` for a remote model is required. This new API is released as an experimental feature in the OpenSearch version 2.16, and only SageMaker, Cohere, and OpenAI are verified as the external servers that support this features.
+
+For information about user access for this API, see [Model access control considerations]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/index/#model-access-control-considerations).
+
+
+For information about connectors and remote models, see [Connecting to externally hosted models]({{site.url}}{{site.baseurl}}/ml-commons-plugin/remote-models/index/). For more details of the connector blurprints for batch predict, see [GitHub docs](https://github.com/opensearch-project/ml-commons/blob/main/docs/remote_inference_blueprints/batch_inference_openAI_connector_blueprint.md)
```

Comment:
I think best action item would be put this blueprint under a sub folder for `batch_prediction` and link to that sub folder. In this way, if we add blueprints for sagemaker and cohere later, CX will still find these in the sub folder. 

Right now we are saying this will work for Sagemaker or Cohere but there's no example for this. And also why customer needs to go another link to get the same blue print what is here?

Reply:
It's fine to have a subfolder for offline actions. But please note that this is the API page showing how this API works. So let's keep this page directly to the point. Using OpenAI as an example to show this API is enough. Other details will be documented elsewhere in blueprints or tutorials.

Reply:
I agree that this API page should be simple. Ideally, I would even remove the prerequisite steps from this API. However, our users have a very disjointed experience when going back and forth from the doc website to the ML repo. I didn't realize blueprints contained the workflow and not just the blueprint itself. I think we should port all ML blueprints and tutorials to the doc repo and have them on the doc site. I can take this on once this version is released. For now, it's fine to leave this API page with the current information.